README.md

```diff
@@ -234,9 +234,16 @@ module.exports = {
 
 All of them are available globally in each Jest test. If you are using ESLint and JavaScript, its recommend to use it in combination with the [eslint-plugin-jest-playwright](https://github.com/playwright-community/eslint-plugin-jest-playwright).
 
-## Put in debug mode
+## Debug mode
 
-Debugging tests can be hard sometimes and it is very useful to be able to pause tests in order to inspect the browser. Jest Playwright exposes a method `jestPlaywright.debug()` that suspends test execution and gives you opportunity to see what's going on in the browser.
+Debugging tests can be hard sometimes and it is very useful to be able to pause tests in order to inspect the browser. There are two ways to put your tests in debug mode:
+
+- Playwright give you [ability](https://playwright.dev/docs/debug/#run-in-debug-mode) to configure the browser for debugging with the `PWDEBUG` environment variable:
+```js
+PWDEBUG=1 jest
+```
+
+- Jest Playwright exposes a method `jestPlaywright.debug()` that suspends test execution and gives you opportunity to see what's going on in the browser.
```

Comment:
How does `jestPlaywright.debug()` interact with `page.pause()`? https://playwright.dev/docs/api/class-page?_highlight=paus#pagepause

It would be great if `jestPlaywright.debug()` also pauses (and allows the user to resume) but I think this should be explicitly explained here. It is good to say here what debugging does both for the Playwright settings, as well as the Jest settings.

Reply:
maybe we can rework that jestPlaywright will use playwright.pause under the hood?

Reply:
@jhildenbiddle `page.pause()` was introduced only in last version. And `jestPlaywright.debug()` does not related to this function. Personally, I don't like `jestPlaywright.debug()` because it's weird and hacky solution to simplify debugging, that came from `jest-puppeteer`

Reply:
@mxschmitt yes we can. But it will be breaking change. I think usage this feature for debugging is much better that `jestPlaywright.debug()`

Reply:
That would be sweet, do you want to create an issue? I can do it if you want but you can probably better explain it. Release it under a 2.0 release then, I think this awesome project has graduated enough to warrant a new major version 👍

Reply:
@thernstig yes. It will be nice if you create an issue)

Reply:
https://github.com/playwright-community/jest-playwright/issues/608